.github/workflows/frontend-release.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Bump New Version
         run: sh scripts/bump.sh
         env:
-          GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
+          RELEASE_TOKEN: ${{ secrets.RELEASE_TOKEN }}
```

Comment:
I think it's still better to have GITHUB_THERE in the name. e.g. `RELEASE_GITHUB_TOKEN` or sth similar, so that it's easier to understand it's related to GitHub, otherwise, people have to dive into the code to find out what is the `RELEASE_TOKEN`.

Reply:
What about `GIT_RELEASE_TOKEN`? I wanted to also change secret to have `GITHUB_RELEASE_TOKEN` but GitHub complains that secrets cannot start with `GITHUB`